elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -19,50 +22,18 @@ $pfe-progress-indicator--border-dark: 4px solid rgba(0, 0, 0, 0.75);
     border-radius: 100%;
     
     display: inline-block;
-    height: 2rem;
+    height: var(--pfe-progress-indicator--height, 2rem);
```

Comment:
For BEM approach this should use uppercase for the name of css properties so: `--pfe-progress-indicator--Height` and `--pfe-progress-indicator--Width`